quickwit-metastore/src/metastore/single_file_metastore.rs

```diff
@@ -691,4 +707,32 @@ mod tests {
             .unwrap();
         assert!(!split.is_empty());
     }
+
+    #[tokio::test]
+    async fn test_single_file_metastore_race_condition() {
```

Comment:
I don't see how this test creates a potential race condition.

Reply:
The test was still unfinished. It has just been updated.